spec/e2e/pbivizNewSpec.js

```diff
@@ -58,7 +58,7 @@ describe("E2E - pbiviz new", () => {
         let template = 'default';
         let visualPath = path.join(tempPath, visualName);
 
-        FileSystem.runPbiviz('new', visualName);
+        FileSystem.runPbiviz('new', visualName, ' -t default1');
```

Comment:
Why do we need two templates now? Can't we force all of new developers to use ES2015.
I think it makes sense not to use old modules anymore.

Reply:
We can't break backward compatibility. Old templates uses in UTs

Reply:
ok